Packs/CommonScripts/Scripts/FormattedDateToEpoch/FormattedDateToEpoch.py

```diff
@@ -1,17 +1,19 @@
 import demistomock as demisto
+import dateparser
 from datetime import datetime, timezone
+from typing import Optional
 
 
-def date_to_epoch(date, formatter):
+def date_to_epoch(date: str, formatter: Optional[str] = None):
     epoch = datetime(1970, 1, 1, tzinfo=timezone.utc)
-    date_obj = datetime.strptime(date, formatter)
+    date_obj = datetime.strptime(date, formatter) if formatter else dateparser.parse(date)
```

Comment:
I think there's a default value to this argument, so it would never be `None` no?

Reply:
You are right, so I'll remove the default value?